google/cloud/internal/oauth2_minimal_iam_credentials_rest_test.cc

```diff
@@ -234,9 +236,9 @@ TEST(MinimalIamCredentialsRestTest, GenerateAccessTokenSuccess) {
 
           EXPECT_THAT(
               request.path(),
-              Eq(absl::StrCat("https://iamcredentials.googleapis.com/v1/",
-                              "projects/-/serviceAccounts/", service_account,
-                              ":generateAccessToken")));
+              Eq(absl::StrCat("https://iamcredentials.", universe_domain,
+                              "/v1/projects/-/serviceAccounts/",
+                              service_account, ":generateAccessToken")));
           std::string str_payload(payload[0].begin(), payload[0].end());
           EXPECT_THAT(str_payload,
                       testing::HasSubstr("\"lifetime\":\"3600s\""));
```

Comment:
nit from before this PR: `using ::testing::HasSubstr;`, and then change these to just `HasSubstr(...)`